### ISSUE TITLE
Changing platform labels in Flutter

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#53204]
[The labels we use for different platforms ](https://github.com/flutter/flutter/labels?q=platform)are not accessible due to the icons in them.

The screen readers are trying to announce the icon descriptions before announcing the name of the label which makes them very hard to comprehend.

The current labels also makes triaging issues harder since manually typing a platform query is very hard with the icons.

**Comment by goderbauer** [comment]
If we do end up changing the labels, all links in https://github.com/flutter/flutter/wiki/Triage need to be updated.

**Comment by nturgut** [comment]
Thanks for the heads up! I'll make sure it is updated.

**Comment by yjbanov** [comment]
I can update all of the Web-related links. We don't have any on the https://github.com/flutter/flutter/wiki/Triage page (we can fix that separately). They are all internal link. I'm happy to update them. Just tell me when.

**Comment by nturgut** [comment]
I changed the Triage page. https://github.com/flutter/flutter/labels?q=platform

@yjbanov Can you change the web triage link?

**Comment by yjbanov** [comment]
Done

**Comment by nturgut** [comment]
> Done

Thanks! Closing the issue.
